modules/ROOT/pages/access-developer-portal-task.adoc

```diff
@@ -1,27 +1,34 @@
-= Accessing the Developer Portal
+= Access the Developer Portal
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API portals listed on an organization's developer portal that are public, or to which you have been granted Portals Viewer access, are visible to you. From the public Anypoint Platform Developer portal, you can access Anypoint Platform APIs to automate Anypoint Platform activities.
+As an administrator, you can view API portals if you have the appropriate access. The public https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform developer portal], enables you to automate Anypoint Platform activities.
+ 
+You can view an API portal in Anypoint Platform if one of the below conditions is true:
 
-In API Manager, from the API Administration page, use the hamburger menu on the right. Alternatively, use the following procedure.
+* The portal is listed in the organization's public developer portal
+* You are granted Portals Viewer access for a portal 
 
-To access the Anypoint Platform Developer Portal:
+== Access API Instance Portal
 
-The Anypoint Platform developer portal is a public portal that you can find by searching on the internet or by clicking the link at the bottom of this page.
+To access an API instance portal:
 
-To access your organization's developer portal, use one of the following methods:
-
-. Go to Access Management > Organization, and click the name of your organization.
-+
-Organization Info appears, showing the Domain Name.
+. Go to *API Manager* > *Administration* tab.
+. Select the API instance that you want to view in the developer portal.
+. From the top-right corner on the API instance page, click *View API in Exchange*.
 +
-. Go to your organization's developer portal using a URL in the following format, substituting domain-name for your organization's domain name:
-+
-`+http://anypoint.mulesoft.com/apiplatform/domain-name/#/portals+`
+The Developer portal for the selected asset is displayed in a new window. You can also share and make the xref:exchange::about-portals.adoc[portal public] in Anypoint Exchange.
+
+== Access the Public Portal of an Organization
+
+To access the public portal of an organization:
+
+. Go to *Exchange* > *Public portal*.
+. From the developer portal page, select the asset that you want to access.
```

Comment:
```suggestion
. From the developer portal page, select the portal that you want to access.
```
GLOBALLY avoid using more than one term to mean one thing: "asset" or "portal" or "instance"?

Reply:
Asset, portal, instance these are all different things.